glib/src/functions.rs

```diff
@@ -238,7 +241,7 @@ pub fn compute_checksum_for_string(
 
 #[cfg(unix)]
 #[doc(alias = "g_unix_open_pipe")]
-pub fn unix_open_pipe(flags: i32) -> Result<(RawFd, RawFd), Error> {
+pub unsafe fn unix_open_pipe(flags: i32) -> Result<(RawFd, RawFd), Error> {
```

Comment:
Should probably return `OwnedFd`?

Reply:
Is that compatible with the `flags` argument that allows setting O_CLOEXEC/FD_CLOEXEC?

Reply:
I think so